aten/src/ATen/native/native_functions.yaml

```diff
@@ -739,15 +739,12 @@
   use_c10_dispatcher: unboxed_only
   supports_named_tensor: True
   variants: function, method
-  dispatch:
-    CPU: _cosh__cpu
-    CUDA: _cosh__cuda
 
 - func: cosh.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
   supports_named_tensor: True
   dispatch:
-    CPU: _cosh_out_cpu
-    CUDA: _cosh_out_cuda
+    CPU: cosh_out
```

Comment:
can be removed

Reply:
Which lines can be removed?

Reply:
you can remove:
dispatch:
    CPU: cosh_out
    CUDA: cosh_out